api/samples/instrace_simple.c

```diff
@@ -203,6 +205,7 @@ instrument_instr(void *drcontext, instrlist_t *ilist, instr_t *where)
     if (drreg_unreserve_register(drcontext, ilist, where, reg_ptr) != DRREG_SUCCESS ||
         drreg_unreserve_register(drcontext, ilist, where, reg_tmp) != DRREG_SUCCESS)
         DR_ASSERT(false);
+    instrlist_set_auto_predicate(ilist, instr_get_predicate(where));
```

Comment:
The client actually doesn't need to reset the predicate--drmgr does it for us automatically. But agreed, this is a little confusing so having a `drmgr_disable_auto_predication()` method would make sense.

Done.